lib/ably/models/delta_extras.rb

```diff
@@ -0,0 +1,38 @@
+module Ably::Models
+  #
+  # @!attribute [r] from
+  #   @return [String] The id of the message the delta was generated from
+  # @!attribute [r] format
+  #   @return [String] The delta format. Only vcdiff is supported as at API version 1.2
+  #
+  class DeltaExtras
+    include Ably::Modules::ModelCommon
+
+    # DeltaExtras attributes
+    # @return [Hash]
+    #
+    attr_reader :attributes
+
+    def initialize(attributes = {})
+      @attributes = IdiomaticRubyWrapper((attributes || {}), stop_at: [:from, :format])
```

Comment:
What do you think about? 
```
attr_reader :from, :format

@from, @format = IdiomaticRubyWrapper((attributes || {}), stop_at: [:from, :format]). values_at(:from, :format)
```

It's way shorter and doesn't introduce and expose `attributes`

Reply:
Fixed